### DISCUSSION TITLE
Implement gradient checkpointing 实现梯度检查点

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#630] [MMEngine]
# Implement gradient checkpointing 
 实现梯度检查点 

[https://github.com/open-mmlab/mmengine/issues/1258](https://github.com/open-mmlab/mmengine/issues/1258)

[https://github.com/open-mmlab/mmengine/issues/1258](https://github.com/open-mmlab/mmengine/issues/1258)

## Task Information

**Repository**：MMEngine

**Difficulty**：Medium

**Points**：40

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：zeyuanyin

**Claim Time**：2023年08月16日

**PR Link**：https://github.com/open-mmlab/mmengine/pull/1319

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by zeyuanyin** [comment]
PR link: https://github.com/open-mmlab/mmengine/pull/1319
